src/blocks/icons/components/Curve.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+import { IconWrapper } from '../IconWrapper';
+import { IconProps } from '../Icons.types';
+
+const Curve: FC<IconProps> = (allProps) => {
```

Comment:
- See if this could be a div
- If yes then remove this component
- If not then confirm from Zee about the naming of this component and ask him to add it to figma icons